File: app/src/main/java/com/telolahy/guitarmemo/ChordsActivity.java
package com.telolahy.guitarmemo;

import android.os.Bundle;
import android.webkit.WebView;
import androidx.appcompat.app.AppCompatActivity;

public class ChordsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_chords_layout);

        setTitle(getString(R.string.chords));

        WebView webView = (WebView) this.findViewById(R.id.webView1);
        webView.getSettings().setBuiltInZoomControls(true);
        webView.loadUrl("file:///android_asset/Cowboy-Chords.jpg");
    }

}

File: app/src/main/java/com/telolahy/guitarmemo/ChordsActivity.kt
package com.telolahy.guitarmemo

import android.os.Bundle
import android.view.View
import android.webkit.WebView
import androidx.appcompat.app.AppCompatActivity

class ChordsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_chords_layout)

        title = getString(R.string.chords)

        val webView = findViewById<View>(R.id.webView1) as WebView
        webView.settings.builtInZoomControls = true
        webView.loadUrl("file:///android_asset/Cowboy-Chords.jpg")
    }
}
